testslide/__init__.py

```diff
@@ -421,7 +421,7 @@ def showwarning(
             else:
                 original_showwarning(message, category, filename, lineno, file, line)
 
-        warnings.showwarning = showwarning
+        warnings.showwarning = showwarning  # type: ignore
```

Comment:
I'm curious why this ignore was needed now (same for `logger_warning` below). Is this one of the cases we could / should add an exception at `mypy.ini` as `ignore_missing_imports = True`?

Reply:
Nono, it's ```testslide/__init__.py:441: error: Cannot assign to a method```

Reply:
Something's odd. Just tried this on master (8833026):

- Remove these 2 `type: ignore`.
- Pyenv with Python 3.8.5.
- `python -m venv venv`
- `. venv/bin/activate`
- `make install_build_deps`
- `make mypy`

and... no errors reported! Not sure what extra you have at your machine to detect this issue, but I'd love to have it on master, lol... I think it is time start using [tox](https://tox.readthedocs.io/en/latest/) and have a file with all dependency versions frozen, which we update every release.